src/pages/ContactPage.vue

```diff
@@ -0,0 +1,119 @@
+<template>
+  <div class="contact-page">
+    <BreadCrumbs :steps="steps" :back-route="{ name: 'Home' }" />
+    <SectionTitle class="contact-page__title">Contact Us</SectionTitle>
+    <div class="contact-page__card">
+      <div class="contact-page__image">
+        <MapImage />
+      </div>
+      <ContactCard class="contact-page__details" />
+      <div class="contact-page__icons">
+        <TweeterIcon class="contact-page__icon" />
+        <FacebookIcon class="contact-page__icon" />
+        <LinkedinIcon class="contact-page__icon" />
+        <InstagramIcon class="contact-page__icon" />
+        <GitIcon class="contact-page__icon" />
+        <BeIcon class="contact-page__icon" />
+        <BasketballIcon class="contact-page__icon" />
+      </div>
+    </div>
+  </div>
+</template>
+
+<script>
+import { reactive, toRefs } from "vue";
+import BreadCrumbs from "../components/common/BreadCrumbs.vue";
+import SectionTitle from "../components/common/SectionTitle.vue";
+import MapImage from "@assets/images/map.svg?component";
+import ContactCard from "../components/common/ContactCard.vue";
+
+import TweeterIcon from "@assets/images/icons/tweeter.svg?component";
+import FacebookIcon from "@assets/images/icons/facebook.svg?component";
+import LinkedinIcon from "@assets/images/icons/linkedin.svg?component";
+import InstagramIcon from "@assets/images/icons/instagram.svg?component";
+import GitIcon from "@assets/images/icons/git.svg?component";
+import BeIcon from "@assets/images/icons/be.svg?component";
+import BasketballIcon from "@assets/images/icons/basketball.svg?component";
+
+export default {
+  setup() {
+    const state = reactive({
+      steps: [{ text: "Contact us", isLink: false }],
+    });
+    return {
+      ...toRefs(state),
+    };
```

Comment:
This becomes a bit confusing I feel, cause it requires keeping in mind the context in which toRefs is being used. 
I'd suggest using just `ref`
```suggestion
    const steps = ref([{ text: "Contact us", isLink: false }]),
    });
    return { steps };
```

Reply:
I've tried a snippet here, and you're right, it gets confusing.